cmd/kubestellar-where-resolver/options/options.go

```diff
@@ -37,17 +43,18 @@ func NewOptions() *Options {
 	logs.Config.Verbosity = config.VerbosityLevel(2)
 
 	return &Options{
-		EspwClientOpts: *clientoptions.NewClientOpts("espw", "access to the edge service provider workspace"),
-		BaseClientOpts: *clientoptions.NewClientOpts("base", "access to all logical clusters as kcp-admin"),
-		Logs:           logs,
+		SpaceMgtClientOpts: *clientoptions.NewClientOpts("space-mgt", "access to the space reference space"),
```

Comment:
Sorry, since I last commented on the wording here we agreed on a different term than I anticipated. We are now calling it the "space management API space".